models/fv_ae_pg_lag_pol_att.go

```diff
@@ -0,0 +1,70 @@
+package models
```

Comment:
name of file should be fv_aepg_lag_pol_att.go

Reply:
I thought we had fixed that. Can you open an issue against autogen for this issue?